### PR TITLE
simplify distribution

### PR DESCRIPTION
**Pull request by mhoffm-aiven**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,14 +44,18 @@ $(GEN_CRD_API_REF_DOCS): $(TOOLS_DIR)/go.mod ## Build gen-crd-api-ref-docs from
...
+		trap 'rm -rf $(TMPDIR)' EXIT \
+		&& cd $(TMPDIR) \
+		&& wget ${OPERATOR_SDK_DL_URL}/operator-sdk_$(GOOS)_$(GOARCH) \
+		&& wget ${OPERATOR_SDK_DL_URL}/checksums.txt \
+		&& wget ${OPERATOR_SDK_DL_URL}/checksums.txt.asc \
+		&& gpg --keyserver keyserver.ubuntu.com --recv-keys 052996E2A20B5C7E \
+		&& gpg -u 'Operator SDK (release) <cncf-operator-sdk@cncf.io>' --verify $(TMPDIR)/checksums.txt.asc \
+		&& grep operator-sdk_${GOOS}_${GOARCH} checksums.txt | sha256sum -c - \
+		&& chmod 750 operator-sdk_$(GOOS)_$(GOARCH) \
+		&& sudo mv operator-sdk_$(GOOS)_$(GOARCH) $(ROOT_DIR)/${OPERATOR_SDK}"
```
Comment (ivan-savciuc) [inline Makefile:58]: Can we do without `sudo`? Some Linux distributions do not have it out of the box?

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: aivenoy/aiven-operator
-  newTag: v0.1.1
+  newName: controller
```
Comment (ivan-savciuc) [inline config/manager/kustomization.yaml:7]: Why do we change it here to default  `controller:latest`?

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
Tested it locally on mac and fresh Linux VM. 

On Mac had to install `coreutils` to add support for `sha256sum`.
```
gpg: key 052996E2A20B5C7E: public key "Operator SDK (release) <cncf-operator-sdk@cncf.io>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: assuming signed data in '/var/folders/gm/9dl7mqbx0y9_q29z19btzshw0000gn/T/tmp.ZPlwFop9/checksums.txt'
gpg: Signature made Thu Sep  9 19:59:50 2021 EEST
gpg:                using RSA key 8613DB87A5BA825EF3FD0EBE2A859D08BF9886DB
gpg: Good signature from "Operator SDK (release) <cncf-operator-sdk@cncf.io>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 3B2F 1481 D146 2380 80B3  46BB 0529 96E2 A20B 5C7E
     Subkey fingerprint: 8613 DB87 A5BA 825E F3FD  0EBE 2A85 9D08 BF98 86DB
bash: sha256sum: command not found
```

It would be nice if we could address this dependency.

File: Makefile (hunk 1 above)
Comment (mhoffm-aiven) [reply Makefile]: yes we can do without; i removed it

File: config/manager/kustomization.yaml (hunk 2 above)
Comment (mhoffm-aiven) [reply config/manager/kustomization.yaml:7]: its expected to be that way in the CI pipeline, also the change was accidentally introduced in the previous PR. Also i think it was autogenerated that way. I think we should revert it to the previous version and not touch it again.

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
Testing `make bundle-build bundle-push`

Getting this error:
```
make bundle-build bundle-push IMG="eu.gcr.io/integral-magnet-301414/aiven-operator:v12"
Makefile:182: warning: overriding commands for target `&'
Makefile:68: warning: ignoring old commands for target `&'
make: *** No rule to make target `bundle-build'.  Stop.
```

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**